modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/info/display/field/DDMFormValuesInfoDisplayFieldProviderImpl.java

```diff
@@ -195,7 +195,11 @@ private Object _sanitizeFieldValue(
 
 		Value value = ddmFormFieldValue.getValue();
 
-		String valueString = value.getString(locale);
+		String valueString = StringPool.BLANK;
+
+		if (value != null) {
```

Comment:
Hi @crodriguezy,

I believe we can optimize a little your solution, once, right now, your proposed fix is keeping the execution of the method `SanitizerUtil.sanitize` but, according to what I understood from your fix, it isn't necessary for the Separator field, correct?

Can you please resend your fix with something like:
```
Value value = ddmFormFieldValue.getValue();

if (value != null) {
    return StringPool.BLANK;
}
```

Thanks in advance.

Reply:
Hi @natocesarrego , I did it that way because I checked, by debugging, that valueStrings equals to "" coming fron not null values would also go through the sanitize method.

Should we check it this way instead?: 

```
Value value = ddmFormFieldValue.getValue();

if (Validator.isNull(value)) {
    return StringPool.BLANK;
}
```
Anyway, in case you prefer the one you proposed (I guess you meant the `value == null `comparison instead of `!=`) please see  https://github.com/natocesarrego/liferay-portal/pull/199

Thanks

Reply:
Hi @crodriguezy,

I checked the `SanitizerUtil.sanitize` method and it intends to remove some specific String patterns from the `valueString` parameter, therefore if this parameter is empty then I don't see need for we invoke this method. What do you think?

Additionally, your understanding regarding my suggestion and its error is correct, sorry for my mistake. 😁 At last, I just checked your new pull and you did exactly what I was thinking. 🙂

Best.

Reply:
Hi @natocesarrego, I agree with you thay we do not need to call the sanitize method for empty and null values. However I've double checked the Validator.isNull() method to see if this is really the way we want to fix it to prevent both null and empty values from going through the sanitize method:

```
Value value = ddmFormFieldValue.getValue();

if (Validator.isNull(value)) {
    return StringPool.BLANK;
}
```
But I'm not sure we should use **_Validator.isNull_** since it checks too many conditions:

> 	/**
> 	 * Returns <code>true</code> if the string is <code>null</code>, meaning it
> 	 * is a <code>null</code> reference, an empty string, whitespace, or the
> 	 * string "<code>null</code>", with or without leading or trailing
> 	 * whitespace.
> 	 *
> 	 * @param  s the string to check
> 	 * @return <code>true</code> if the string is <code>null</code>;
> 	 *         <code>false</code> otherwise
> 	 */

Anyway I've also seen that you already push my other PR to Brian.

Thanks!

Reply:
Hi @crodriguezy,

The last pull you sent to me was great, then I forwarded it to Brian Chan. 🙂 

Thanks.